lib/srv/forward/sshserver.go

```diff
@@ -861,31 +938,62 @@ func (s *Server) rejectChannel(chans <-chan ssh.NewChannel, errMessage string) {
 	}
 }
 
+// logErrorToChannel writes an error message over a channel and logs a more
+// detailed message locally.
+func (s *Server) logErrorToChannel(ch ssh.Channel, err error, format string, args ...any) {
+	s.log.WithError(err).Errorf(format, args...)
+	s.stderrWrite(ch, fmt.Sprintf(format, args...))
```

Comment:
If it turns out that we can and want to write to stderr on a channel for errors:
```suggestion
	f := fmt.Sprintf(format, args...)
	s.log.WithError(err).Error(f)
	s.stderrWrite(ch, f)
```